src/applications/ivc-champva/10-10D/containers/ConfirmationPage.jsx

```diff
@@ -48,9 +74,12 @@ export function ConfirmationPage(props) {
     data: form.data,
     disableLinks: true,
     heading,
+    showRequirementHeaders: false,
+    requiredDescription: '',
     optionalWarningHeading: <>{optionalWarningHeading}</>,
     requiredWarningHeading: <>{requiredWarningHeading}</>,
     showMail: true,
+    mailPreamble,
```

Comment:
Adjusting parameters that change the display of the `MissingFileOverview` component